src/include/binder/bind_node_visitor.h

```diff
@@ -57,6 +58,7 @@ class BindNodeVisitor : public SqlNodeVisitor {
   void Visit(expression::CaseExpression *expr) override;
   // void Visit(const expression::ConstantValueExpression *expr) override;
   void Visit(expression::TupleValueExpression *expr) override;
+  concurrency::Transaction *txn = nullptr;
```

Comment:
@apavlo is right. Please write down the reason why you declare it as member variable. Also, please try to make member variables private.